.gitignore

```diff
@@ -5,6 +5,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+.DS_Store
```

Comment:
I think this should go in your global gitignore rather than in the project-specific gitignore (wich is supposed to stay OS-agnostic), no?

Reply:
This is much easier for contributors though

Reply:
It's your call, really. If you see this as a valuable addition, why not :-)

Reply:
On second thought keeping it OS agnostic is good. I guess you're right this should go in global.